featuretools_tsfresh_primitives/tests/test_primitives.py

```diff
@@ -1,153 +1,74 @@
-    "MaxLangevinFixedPoint": featuretools_tsfresh_primitives.MaxLangevinFixedPoint,
-    "Maximum": featuretools_tsfresh_primitives.Maximum,
-    "Mean": featuretools_tsfresh_primitives.Mean,
-    "MeanAbsChange": featuretools_tsfresh_primitives.MeanAbsChange,
-    "MeanChange": featuretools_tsfresh_primitives.MeanChange,
-    "MeanSecondDerivativeCentral": featuretools_tsfresh_primitives.MeanSecondDerivativeCentral,
-    "Median": featuretools_tsfresh_primitives.Median,
-    "Minimum": featuretools_tsfresh_primitives.Minimum,
-    "NumberCrossingM": featuretools_tsfresh_primitives.NumberCrossingM,
-    "NumberCwtPeaks": featuretools_tsfresh_primitives.NumberCwtPeaks,
-    "NumberPeaks": featuretools_tsfresh_primitives.NumberPeaks,
-    "PartialAutocorrelation": featuretools_tsfresh_primitives.PartialAutocorrelation,
-    "PercentageOfReoccurringDatapointsToAllDatapoints": featuretools_tsfresh_primitives.PercentageOfReoccurringDatapointsToAllDatapoints,
-    "PercentageOfReoccurringValuesToAllValues": featuretools_tsfresh_primitives.PercentageOfReoccurringValuesToAllValues,
-    "Quantile": featuretools_tsfresh_primitives.Quantile,
-    "RangeCount": featuretools_tsfresh_primitives.RangeCount,
-    "RatioBeyondRSigma": featuretools_tsfresh_primitives.RatioBeyondRSigma,
-    "RatioValueNumberToTimeSeriesLength": featuretools_tsfresh_primitives.RatioValueNumberToTimeSeriesLength,
-    "SampleEntropy": featuretools_tsfresh_primitives.SampleEntropy,
-    "Skewness": featuretools_tsfresh_primitives.Skewness,
-    "SpktWelchDensity": featuretools_tsfresh_primitives.SpktWelchDensity,
-    "StandardDeviation": featuretools_tsfresh_primitives.StandardDeviation,
-    "SumOfReoccurringDataPoints": featuretools_tsfresh_primitives.SumOfReoccurringDataPoints,
-    "SumOfReoccurringValues": featuretools_tsfresh_primitives.SumOfReoccurringValues,
-    "SumValues": featuretools_tsfresh_primitives.SumValues,
-    "SymmetryLooking": featuretools_tsfresh_primitives.SymmetryLooking,
-    "TimeReversalAsymmetryStatistic": featuretools_tsfresh_primitives.TimeReversalAsymmetryStatistic,
-    "ValueCount": featuretools_tsfresh_primitives.ValueCount,
-    "Variance": featuretools_tsfresh_primitives.Variance,
-    "VarianceLargerThanStandardDeviation": featuretools_tsfresh_primitives.VarianceLargerThanStandardDeviation,
-}
-
-PARAMETERS = ComprehensiveFCParameters()
-
-
-@fixture(scope='session')
-def entityset():
-    return ft.demo.load_mock_customer(return_entityset=True)
-
+from featuretools_tsfresh_primitives import comprehensive_primitives
 
-@fixture
-def parameters():
-    dirname = os.path.dirname(__file__)
-    file = os.path.join(dirname, 'test_primitives.json')
-    return json.load(open(file, 'r'))
+BLACKLIST = [
+    # when a partial autocorrelation has a lag of zero,
+    # an error is raised from `tsfresh.feature_extraction.extract_features`
+    'PARTIAL_AUTOCORRELATION(transactions.amount, lag=0)'
```

Comment:
If this primitive will always error (I think this is the case?), perhaps we should not include it in supported primitives

Reply:
Specifically the `lag=0` version of partial autocorrelation

Reply:
The primitive doesn't error when the lag is greater than zero -- only errors specifically at zero. Not sure why this parameter value would be an option in `ComprehensiveFCParameters`.

Reply:
Right.  So what do you think of, by default, not including the instance of PartialAutocorrelation with lag=0 when we return `supported_primitives`

Reply:
It might make a misalignment between `ComprehensiveFCParameters` and `comprehensive_primitives`. Maybe we can wrap `ComprehensiveFCParameters` in a function that filters out nonfunctional parameter value.

Reply:
I was confused, `supported_primitives` returns the primitive classes, not instances of the classes with specific parameters.  I see what you mean about misalignment if we leave out non-functional parameter values from `comprehensive_primitives`

Reply:
Yes, that's correct. If we remove the non-function parameter values from the start, then I don't think we should have any problems of misalignment.